index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+        </div>
+     </section>
+
+     <section id="about-me">
+        <h1>About me</h1>
```

Comment:
- [ ] Kindly implement the [required changes](https://github.com/JansueT/Portfolio/pull/2/files#r1038809901) from the previous reviewer. 

![Screenshot from 2022-12-04 01-19-33](https://user-images.githubusercontent.com/60083437/205458104-ca6b200f-1b20-4c36-96a9-4d37e49cf841.png)

Reply:
- [ ] Good job of making some improvement required in the previous review but I can see that your about has a `border` that is not on real design , please remove that border on your about section to match the figma design 

Figma
![figma21](https://user-images.githubusercontent.com/88894525/206862754-fcc6f4c1-a7f1-482f-8f91-4bca54c8d08d.png)

your design
![removeborder](https://user-images.githubusercontent.com/88894525/206862837-e47cc137-c189-4641-90d2-dbe88cdfb4d4.png)